csbuild/tools/cpp_compilers/clang_cpp_compiler.py

```diff
@@ -68,9 +76,15 @@ def _getArchitectureArgs(self, project):
 	####################################################################################################################
 
 	def _getArchTarget(self, project):
+		if not self._targetSuffix:
+			return None
+
 		# When necessary fill in the architecture name with something clang expects.
-		architecture = {
+		targetPrefix = {
```

Comment:
Commenting here just because I don't like you.

Dr. Evazan doesn't like you either.

Reply:
I'll be careful.

Reply:
You'll be dead!

Reply:
This little one (referring to myself) isn't worth the effort.  Come, let me get you something.

Reply:
You don't want to get me death sticks. You want to go home and rethink your life.

Reply:
I want to go home and rethink my life...